### ISSUE TITLE
Nanotrasen Shipping Containers sprite

### ISSUE THREAD

**Issue by Generubi** [tgstation/tgstation#86443]
### Client Version:

515.1640

### Issue Summary:

The containers make it possible to hide a whole crate (with possibly bodies) behind it, which seems not intended and allows for being round removed without possibility of even the most devoted paramedics finding you.
![image](https://github.com/user-attachments/assets/1e72cea5-da4e-4d46-b666-ab595107c319)
![image](https://github.com/user-attachments/assets/3a22dcd1-3236-4de8-8cbe-e346c571f1ef)

![image](https://github.com/user-attachments/assets/808914c9-ad07-46da-a318-cb0e1b133734)
if you are out of a crate your dead player sprite is barely visible

### Round ID:

236572

### Reproduction Steps:

1. Go to Tramstation
2. Go to the maintenance left of xenobio
3. see the hulking sprite of the containers

**Comment by Sadboysuss** [comment]
it would probably be good if it had collision extended to the whole sprite, or just don't make sprites that extend over their tiles

**Comment by Generubi** [comment]
> it would probably be good if it had collision extended to the whole sprite, or just don't make sprites that extend over their tiles

Yeah, that is what i thought would fix it!! a colission appropiate with the size!
